ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -0,0 +1,80 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-aws-414
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-kube-burner
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      WORKER_REPLICA_COUNT: "20"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: workers-scale
+    workflow: openshift-qe-ocp-qe-perfscale-ci-tests
```

Comment:
@paigerube14 Do we want to use this workflow for all perfscale-ci-tests or instead list all tests at this high level config?
The reason I ask is, if any of the tests in that workflow fail, the remaining tests won't run. But, if we list all tests we gurantee running all tests independent of each other's test result.

Reply:
Will it run the rest of the tests if one fails? I've never seen that occur. I've only seen if one test fails it'll go to the post

Reply:
If each test is mentioned separately, they run independently.

Reply:
Another option I can think of is, changing `openshift-qe-ocp-qe-perfscale-ci-tests` to chain from workflow. That will also allow all tests to run independent of each other's result. And that will also allow us to keep high level config clean. Something worth trying IMO.

Reply:
think we should investigate this in a different PR. want to try to get this in